### ISSUE TITLE
tlatools doesn't accept absolute spec paths on Windows

### ISSUE THREAD

**Issue by hwayne** [tlaplus/tlaplus#424]
When running

```
java -jar D:\path\to\tla2tools.jar -config D:\path\to\config.cfg D:\Path\To\Spec\.tla
```

I get the following error:

```
See the User Output or TLC Console for clues to what happened.
The exception was a tlc2.tool.ConfigFileException
: TLC encountered the following error when trying to read the configuration file D:\Path\To\Spec\.tla
File not found.
```

If I run start in `D:\` and run

```
java -jar D:\path\to\tla2tools.jar -config D:\path\to\config.cfg .\Path\To\Spec\.tla
```

Then everything works as expected. I get the same results using both `-jar tla2tools.jar` and `-cp tla2tools.jar tlc2.TLC`.

This is on the 2/2 nightly. I have not attempted to reproduce on Linux.

**Comment by quaeler** [comment]
@hwayne questions!
* in your examples, your TLA file actually has a basename, ya? (it's not `.tla` but rather `Something.tla`, i mean)
* is this something that has broken for you, or has never worked as far as you know?

**Comment by hwayne** [comment]
* Yeah, it has a basename. Should have written `path\to\spec.tla`
* Paths used to not work for any OS, see https://github.com/tlaplus/tlaplus/pull/420. Now relative paths work on Windows but absolute paths do not.

**Comment by quaeler** [comment]
This has been addressed with 0abc64e
